code/game/gamemodes/clock_cult/clock_structures.dm

```diff
@@ -352,19 +352,19 @@
 				else
 					mech_damage_cycle++
 			setDir(get_dir(get_turf(src), get_turf(target)))
-	if(!target)
+	else
```

Comment:
This wasn't an else so it'd immediately acquire a new target if the old one wasn't valid, intentional?